modules/exporer-remix-solidity-services.nix

```diff
@@ -0,0 +1,87 @@
+{ pkgs, lib, nodes, options, config, ... }:
```

Comment:
typo in filename? exporer? should be explorer maybe?

Reply:
👩‍🏫